### ISSUE TITLE
AEX-2: Use responses with errors instead of separate error method

### ISSUE THREAD

**Issue by davidyuk** [aeternity/AEXs#44]
> `error`: used to communicate any error occurred. Embedded data field in each error object MUST contain the request that triggered it.

JSON RPC supports error objects as part of the [response object](https://www.jsonrpc.org/specification#response_object), so I think it is not necessary to introduce an additional `error` method and force to pass all error through it ("used to communicate any error occurred"). In which cases `error` method is needed?

**Comment by shekhar-shubhendu** [comment]
It is a JSON-RPC response object. I'll move it under a different heading and make the text more clear.

### Proposal
```json
{
      "jsonrpc": "2.0",
      "error": {
          "code": 1,
          "data": {
              "request": {}
          },
          "message": "Transaction verification failed"
      },
      "id": null
  }

```

### Standard JSON-RPC Error Response Object
```json
{
	"jsonrpc": "2.0",
	"error": {
		"code": -32700,
		"message": "Parse error"
	},
	"id": null
}
```

**Comment by davidyuk** [comment]
Currently, it is in the same list with `ping/pong` that is definitely methods, please explain it better in AEX.

**Comment by davidyuk** [comment]
Solved in #57
